.circleci/config.yml

```diff
@@ -59,9 +59,17 @@ commands:
       - run:
           name: basic install
           command: |
-            apt-get update
+            apt-get update --fix-missing
```

Comment:
Do we need this?

Reply:
Sometimes apt loses its mind during updates. An update with --fix-missing solves many of these issues, when possible It rebuilds the index